extra_data/reader.py

```diff
@@ -1454,13 +1454,19 @@ def RunDirectory(
         creating child processes is not allowed (e.g. in a daemonized
         :class:`multiprocessing.Process`).
     """
-    files = [f for f in os.listdir(path) if f.endswith('.h5')]
+    files = [f for f in os.listdir(path)
+             if f.endswith('.h5') and ('overview' not in f.lower())]
```

Comment:
Should we make that condition `f.lower() != 'overview.h5'`? I know it's not likely to have that name in a data aggregator file, but just in case 😁 

```suggestion
             if f.endswith('.h5') and (f.lower() != 'overview.h5')]
```